packages/cli/generators/model/index.js

```diff
@@ -82,20 +82,17 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
       },
     ];
 
-    return this.prompt(prompts).then(props => {
-      if (!props.base) props.base = baseOptions[0];
-      Object.assign(this.artifactInfo, props);
+    const answers = await this.prompt(prompts);
```

Comment:
Never knew this would work; I thought `this.prompt()` never actually returned a promise. TIL'ed